### PR TITLE
Fix for christinamin9-ancientromancivilisation.weebly.com

### PR DESCRIPTION
**Pull request by ghost**
Changes:
- Turned the werid noisy background behind the text to `darkreader-neutral-background`

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -1696,6 +1696,16 @@ INVERT
 
 ================================
 
+christinamin9-ancientromancivilisation.weebly.com
+
+CSS
+.landing-page #main-wrap, .tall-header-page #main-wrap, .short-header-page #main-wrap, .no-header-page #main-wrap {
+    background-image: unset !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1703]: `unset` -> `none`.

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -1696,6 +1696,16 @@ INVERT
 
 ================================
 
+christinamin9-ancientromancivilisation.weebly.com
+
+CSS
+.landing-page #main-wrap, .tall-header-page #main-wrap, .short-header-page #main-wrap, .no-header-page #main-wrap {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1702]: Each selector on a new line.

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (AndrewDaGuy) [reply src/config/dynamic-theme-fixes.config]: Done 👍

**Review by Gusted — APPROVED** [review]
Thank you

**Pull request merged**